packages/api-explorer/src/components/DocMarkdown/DocComponents/Code.tsx

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Is [this](https://styled-components.com/docs/advanced) how we want to style APIX components? I am not very familiar with styled components but it seems to me that it would eliminate the need for a styled version of every component. It would also make it very easy to provide the ability to switch between say a dark and a light theme, or even allow users to provide their own theme.

cc @lukelooker

Reply:
Thanks for your work on this @miketorosian. I echo this question and direct it to @lukelooker and @jhardy: an end goal of the implementation should be a something where theme overrides are defined in one place, hopefully similar to the way custom themes will be working throughout the rest of components. I think that will/should happen "automatically" for API explorer as an extension as we extend theming beyond embed scenarios.  What's the correct pattern to support that automatic pass-down from whatever Looker's theme happens to be active?

Reply:
@josephaxisa - the above tooltip issue has been addressed.